cpp/src/io/parquet/reader_impl.cu

```diff
@@ -1060,12 +1057,13 @@ void reader::impl::decode_page_headers(hostdevice_vector<gpu::ColumnChunkDesc>&
     page_count += chunks[c].max_num_pages;
   }
 
-  chunks.host_to_device(stream);
-  gpu::DecodePageHeaders(chunks.device_ptr(), chunks.size(), stream);
-  pages.device_to_host(stream, true);
+  chunks.host_to_device(_stream);
+  gpu::DecodePageHeaders(chunks.device_ptr(), chunks.size(), _stream);
+  pages.device_to_host(_stream, true);
 }
 
-void decompress_check(device_span<decompress_status const> stats, rmm::cuda_stream_view stream)
+inline void decompress_check(device_span<decompress_status const> stats,
```

Comment:
Always put free function in source files into anonymous namespace.

Reply:
Done